src/v2/guide/components-props.md

```diff
@@ -25,7 +25,29 @@ Vue.component('blog-post', {
 
 Again, if you're using string templates, this limitation does not apply.
 
-## Static and Dynamic Props
+## Prop Types
+
+So far, we've only seen props listed as an array of strings:
+
+```js
+props: ['title', 'likes', 'isPublished', 'commentIds', 'author']
+```
+
+Usually though, you'll want every prop to be a specific type of value. In these cases, you can list props as an object, where the properties' names and values contain the prop names and types, respectively:
```

Comment:
Isn't it "to be _of_ a specific type of value?"

Reply:
I'm not sure on grammatical correctness, but "to be a" is more common than "to be of a". The sentence reads fine to me.

Reply:
Yeah, not a native speaker myself, just drawing my inspiration from the infamous "Argument of type 'string | number' is not assignable" error. Then again, it's very minor and can be safely ignored if need be.

Reply:
@phanan I think if we didn't have "of value" at the end I agree that "of type" would sound better, but as it is sounds good to me. 🙂 Thanks for the review!